src/utils/reportCalculations.ts

```
@@ -433,6 +433,34 @@ export const calculateEmployeeRoleSummariesDaily = (shiftsToProcess: Shift[]): E
       const key = `${procShift.employee.id}-${procShift.role.name}`;
       let existing = summaries.get(key);
 
+      // --- Find active basePayRate from RoleConfig ---
+      const shiftDate = parseISO(procShift.date);
+      let activePayRateConfig: RoleConfig | null = null;
+      if (procShift.role && procShift.role.configs) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)